website/content/docs/agent/template.mdx

```diff
@@ -171,3 +196,45 @@ using the certificates `validTo` field.
 
 This does not apply to certificates generated with `generate_lease: true`. If set
 Vault Agent template will apply the non-renewable, leased secret rules.
+
+## Example
```

Comment:
Should this header be updated to be more specific than "Example"?

Reply:
Updated.